docs/content/en/_index.html

```diff
@@ -4,150 +4,826 @@
+                            <li class="updates__inner-item updates__inner-item--decor">External</li>
+                        </ul>
+                        <h3 class="updates__list-title">Lightning Talk: Falco, Tracee and Tetragon: eBPF Runtime Observability and Security</h3>
+                        <time class="updates__datetime" datetime="2024-03-28">28 Mar, 2024</time>
+                    </a>
+                </li>
+                <li class="updates__item">
+                    <a href="https://www.youtube.com/watch?v=SiQm6N3ucyc" target="_blank" rel="noreferrer noopener">
+                        <ul class="updates__inner-list">
+                            <li class="updates__inner-item updates__inner-item--green-grass">Tutorial</li>
+                            <li class="updates__inner-item updates__inner-item--decor">External</li>
+                        </ul>
+                        <h3 class="updates__list-title">Restrict Access to Secure Files with Tetragon | eBPF Runtime Enforcement</h3>
+                        <time class="updates__datetime" datetime="2024-02-22">22 Feb, 2024</time>
+                    </a>
+                </li>
+                <li class="updates__item">
+                    <a href="https://www.youtube.com/watch?v=MObLmvBeu00" target="_blank" rel="noreferrer noopener">
+                        <ul class="updates__inner-list">
+                            <li class="updates__inner-item updates__inner-item--cornflower">Tutorial</li>
+                            <li class="updates__inner-item updates__inner-item--decor">External</li>
+                        </ul>
+                        <h3 class="updates__list-title">eBPF for Runtime Enforcement | Tetragon Introduction and Overview</h3>
+                        <time class="updates__datetime" datetime="2024-02-21">21 Feb, 2024</time>
+                    </a>
+                </li>
+                <li class="updates__item">
+                    <a href="https://docs.google.com/document/d/1BFMJLdtisiCSLfMct0GHof_ioL-5QVNLEaeMSlk_7Eo/edit?usp=sharing" target="_blank" rel="noreferrer noopener">
+                        <ul class="updates__inner-list">
+                            <li class="updates__inner-item updates__inner-item--cornflower">Article</li>
+                            <li class="updates__inner-item updates__inner-item--decor">External</li>
+                        </ul>
+                        <h3 class="updates__list-title">Tetragon community Call</h3>
+                        <time class="updates__datetime" datetime="2024-02-20">20 Feb, 2024</time>
+                    </a>
+                </li>
+                <li class="updates__item">
+                    <a href="https://cilium.io/blog/2024/02/02/cilium-at-kube-con-eu-2024/" target="_blank" rel="noreferrer noopener">
+                        <ul class="updates__inner-list">
+                            <li class="updates__inner-item updates__inner-item--cornflower">Conference</li>
```

Comment:
any reason we don't have the 
`<li class="updates__inner-item updates__inner-item--decor">External</li>` here?